misc/qlog-adapter.py

```diff
@@ -27,10 +27,21 @@
 
 def handle_packet_received(events, idx):
     frames = []
+    acked = []
     for i in range(idx+1, len(events)):
         ev = events[i]
         if ev["type"] == "packet-prepare" or ev["type"] in QLOG_EVENT_HANDLERS:
             break
+
+        # An ACK frame can't be re-composed in an iteration. Continue buffering
+        # the ACK blocks until all the blocks are processed.
+        if ev["type"] == "ack-block-received":
+            acked.append([ev["ack-block-begin"],ev["ack-block-end"]])
+            continue
+        elif ev["type"] == "quictrace-recv-ack-delay":
```

Comment:
```suggestion
        elif ev["type"] == "ack-delay-receive":
```

Rename this PROBE as well

Reply:
Good point! will do.

Reply:
Done: d8ed2fa